src/model.js

```diff
@@ -159,7 +158,7 @@ export default class Model {
         // add new pages
         this.setPages(
             this.getPages().concat(addedPages.map(url => {
-                this.pushChangeAdd({type: 'page', url});
+                this.pushChangeToAddedGroup({type: 'page', url});
```

Comment:
'object short notation' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).